cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -20,6 +20,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/go-git/go-git/v5"
```

Comment:
very minor, but what made you reach for this vs just parsing git output? Not a big deal, but increases binary size by 3mb:

```
zoekt on  master [$] ❄️ zoekt 
❯ go build -o master ./cmd/zoekt-sourcegraph-indexserver 

zoekt on  master [$?] ❄️ zoekt 
❯ git checkout j/sgfake-branch-config 
branch 'j/sgfake-branch-config' set up to track 'origin/j/sgfake-branch-config'.
Switched to a new branch 'j/sgfake-branch-config'

zoekt on  j/sgfake-branch-config [$?] ❄️ zoekt 
❯ go build -o sgfake-branch-config ./cmd/zoekt-sourcegraph-indexserver 

zoekt on  j/sgfake-branch-config [$?] ❄️ zoekt 
❯ du -h sgfake-branch-config master 
31M	sgfake-branch-config
28M	master
```